zppy/templates/postprocess_tc.bash

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
+{% include 'slurm_header.sh' %}
+
+{{ environment_commands }}
+
+# Turn on debug output if needed
+debug={{ debug }}
+if [[ "${debug,,}" == "true" ]]; then
+  set -x
+fi
+
+# Script dir
+cd {{ scriptDir }}
+
+# Get jobid
+id=${SLURM_JOBID}
+
+# Update status file
+STARTTIME=$(date +%s)
+echo "RUNNING ${id}" > {{ prefix }}.status
+
+# A Bash script to post-process E3SM 6 hourly (h2) instantaneous output to generate a text file storing Tropical Cyclone tracks
+# tempestremap and tempestextremes are built in e3sm-unified from version 1.5.0.
+
+source /global/common/software/e3sm/anaconda_envs/load_latest_e3sm_unified_cori-haswell.sh
+
+#For typical EAM v2 ne gp2 grids.
+start="{{ '%04d' % (year1) }}"
+end="{{ '%04d' % (year2) }}"
+caseid="{{ case }}"
+drc_in={{ input }}/{{ input_subdir }}
+# Warning: tempest-remap can only write grid file on SCRATCH space.
+# The result files will be moved to another path at the end.
+result_dir_fin={{ output }}/post/atm/tc # Directory will be {{ output }}/post/atm/tc/tc-analysis
+mkdir -p $result_dir_fin
+result_dir={{ scratch }}/tc-analysis/
+
+res=30               # res = 30/120 for ne30/ne120 grids
```

Comment:
Line 38 and 39 specified for ne30 pg2 grids. If both were specified in user config?